pyshelf/search/metadata.py

```diff
@@ -1,11 +1,15 @@
-from elasticsearch_dsl import String, Nested, Boolean, DocType
+from elasticsearch_dsl import String, Nested, Boolean, DocType, tokenizer, analyzer
+
+
+# Required for case sensitivity
+metadata_analyzer = analyzer("metadata_analyzer", tokenizer=tokenizer("keyword"))
```

Comment:
Discussed.  I misread and misunderstood the code.  This makes it case SENSITIVE, because by default elasticsearch is case insensitive.  neato.  I guess we are going to add it to the "name" as well